### ISSUE TITLE
🗃️ Archive pages - Kill remaining .ASPX pages

### ISSUE THREAD

**Issue by jeoffreyfischer** [SSWConsulting/SSW.Website-v1-archive#59]
**Description**
1. Check with Matt W. to confirm whether we can kill those pages.
2. Once approved, zz the pages.

List of pages
- [x] CacheManagement
- [x] Company/ConfirmEMailSent.aspx (search code repo - we think this could be an end screen for searching for a lead)
- [x] Demos (both links 404, safe to zz)
- [x] Download (As per my conversation with @adamcogan kill this route but keep /download/padfiles)
- [x] ErrorPage
- [x] FolderBrowser (kill if blocked by authentication)
- [x] Hosting (they all redirect to our azure consulting page)
- [x] MenuMap (is a sitemap, but we already have a sitemap)
- [x] PowerSessions (safe to zz)
- [x] Shop (removed because we are planning to add a shopping page to the v3 site)
- [x] TestMockup (Last relevant edit was in 2011, safe to zz)
- [x] Video (safe to zz, we have videos on youtube, other page references Silverlight)
- [x] zsMaintenance (pages don't have code-behind files. The testimonials in this route are no longer relevant vecause we have testimonials on the live site)
kept
- [x] Diagnostics (Migrate available pages)
